finagle-http/src/main/scala/com/twitter/finagle/Http.scala

```diff
@@ -268,6 +268,13 @@ object Http extends Client[Request, Response] with HttpRichClient
     def withCompressionLevel(level: Int): Client =
       configured(param.CompressionLevel(level))
 
+
+    /**
+     * Enable the collection of HTTP specific metrics. See [[http.filter.StatsFilter]].
+     */
+    def enableHttpStats(): Client =
```

Comment:
Fair enough. I choose enable because there are no arguments passed in. Just like noFailFast.